enhancements/monitoring/user-workload-monitoring.md

```diff
@@ -136,6 +136,26 @@ As a service owner, I'd like to configure alerting rules or recording rules for
 
 As a service owner, I'd like to view configured and fired alerts in the dev perspective of OpenShift console.
 
+### US11
+
+As a member of the operations team, I’d like to get notified if the Prometheus servers are acting strange.
```

Comment:
Strange?

Maybe `unexpectedly`

Reply:
sounds good :+1: will change, it was taken 1:1 from the initial PM research document.

Reply:
fixed